app/containers/Settings/Settings.jsx

```diff
@@ -1,16 +1,27 @@
 // @flow
 import React, { Component } from 'react'
-import { map, reject } from 'lodash'
 import fs from 'fs'
 import storage from 'electron-json-storage'
 
-import Delete from 'react-icons/lib/md/delete'
 import NetworkSwitch from '../App/Sidebar/NetworkSwitch'
 import { recoverWallet } from '../../modules/generateWallet'
 
-import Button from '../../components/Button'
-import { EXPLORERS, MODAL_TYPES, CURRENCIES } from '../../core/constants'
+import Panel from '../../components/Panel'
+import SelectInput from '../../components/Inputs/SelectInput'
+import UnderlinedHeader from '../../components/Headers/UnderlinedHeader'
+import SettingsItem from '../../components/Settings/SettingsItem'
+import SettingsLink from '../../components/Settings/SettingsLink'
+import WalletRecoveryPanel from '../../components/Settings/WalletRecoveryPanel'
+import {
+  EXPLORERS,
+  MODAL_TYPES,
+  CURRENCIES,
+  ROUTES
+} from '../../core/constants'
 import themes from '../../themes'
+import styles from './Settings.scss'
```

Comment:
The file is currently called `settings.scss` you should change its name though instead of this import